### PR TITLE
Added Counterspell Phoenix

### PR DESCRIPTION
**Pull request by pizzalover125**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/phoenix**](https://counterspell-f3et1qwi1.hackclub.dev/phoenix) |
|:-:|
|<a href="https://counterspell-f3et1qwi1.hackclub.dev/phoenix">Screenshot of /phoenix</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/VDVui_sup0HXmEeac339ertn7qrwtXkIJICbtmtIQtE0CU5cX40oozHxsqHuaXz6z1pF_p1JrII920YSvjI2vA.png">(view full size)</a> |







Commit ac1690c09a22f613b20dca795761cf5ce15c6b17 (https://counterspell-f3et1qwi1.hackclub.dev).

File: components/Phoenix/Guilds.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        </Tilt>
+      </div>
+
+      <div className="flex justify-center leading-[3rem] tracking-wider mx-6">
+        <div className="max-w-6xl px-6 py-6 mt-6 border-4 border-dashed border-pink">
+          <p className="text-3xl neuebit">
+            {/* TODO: Change `Example City` to the name of your city */}
+            You can join the guild you identify with for exclusive swag and
+            badges! You also get to compete against other guilds in friendly
+            activities/challenges throughout Counterspell Example City!
```
Comment (samdev-7) [inline components/Phoenix/Guilds.js:50]: You might want to change this!

File: components/Phoenix/Steps.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+              <StepCard
+                img="/photos/7.png"
+                text="Ship what you made and share it with the rest of the world!"
+              />
+            </div>
+          </div>
+          <div className="py-10 mx-6 space-y-4 text-xl uppercase retro">
+            <p>
+              {/* TODO: Change `Example City` to the name of your city */}
+              Counterspell Example City will go on for{" "}
```
Comment (samdev-7) [inline components/Phoenix/Steps.js:38]: You might want to change this!

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Almost there! Some quick fixes.

**Comment by pizzalover125** [comment]
Let me know what else I need to change :)

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**